### DISCUSSION TITLE
Properties shortcut bug.

### DISCUSSION THREAD

**Discussion by reujoal** [JasonLautzenheiser/trizbort#564] [Issues / Bugs] [answered]
This is pretty minor, but is still frustrating as it slows my workflow(I work with a lot of large maps), when copying descriptions into my game world.

So it used to be that when you double-click a room, it opens up the properties in the "description" tab. For some reason this has changed to instead open in the "objects" tab.

I've looked through all the settings and everything, and there doesn't seem to be anyway to change this back.

Looking through the documentation, it says that Alt-E is a shortcut to switch to descriptions tab. Testing this, Alt-E is actually the shortcut for the objects tab, and Alt-O is the shortcut for the descriptions tab. 
As far as I can see, this is a bug. The shortcuts have somehow been swapped which I assume also affects when opening properties, resulting in it opening in the objects tab, rather than the descriptions tab. 

The strange part is, I tried downloading older versions and the problem still exists in those versions.

EDIT - SO I figured out that by using the shortcuts, I can set the default way it opens. The only trick is, the shortcuts are still swapped. So despite Alt-E taking me to objects, it still sets the default on open to description. Using the Alt-O shortcut takes me to descriptions, but sets the default on open to objects. It's kinda confusing but I hope that made sense...

**Comment by JasonLautzenheiser** [comment]
That's strange....thanks for reporting this...I'll take a look

**Comment by JasonLautzenheiser** [comment, marked as answer]
This has been fixed in a recent commit....will be in the next release
